File: takeMeFront/takeMeFront/src/types/servicePackage.ts
/**
 * 服务套餐配置（助餐/助洁/助医等）
 */
export interface servicePackage {
  id: number
  type: 0 | 1 | 2 | 3 | 4 // 服务类型，和Order.serviceType一致
  name: string            // 套餐名称（如“营养套餐A”）
  price: number           // 套餐价格
  description: string     // 套餐说明
  status: 0 | 1           // 0=下架 1=上架
}

File: takeMe/pojo/src/main/java/com/me/entity/ServicePackage.java
package com.me.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@TableName("service_package")
public class ServicePackage {

    @TableId(type = IdType.AUTO)
    private Long id;

    private String name;
    private Integer type;// 0shop 1clean 2meal 3medical 4 companion//0 = 代购 1 = 助洁 2 = 助餐 3 = 助医 4 = 陪伴
    private Integer price;
    private String description;
    private String image;
    private Integer status;

    private LocalDateTime createTime;
}